bundle/openshift/wavefront-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,291 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "wavefront.com/v1alpha1",
+          "kind": "WavefrontProxy",
+          "metadata": {
+            "name": "example-wavefrontproxy",
+            "namespace": "wavefront"
+          },
+          "spec": {
+            "token": "\u003cTOKEN\u003e",
+            "url": "\u003chttps://YOUR_CLUSTER.wavefront.com/api/\u003e",
+            "image": "registry.connect.redhat.com/wavefronthq/proxy:5.7",
+            "openshift": true,
+            "storageClaimName": "wavefront-proxy-storage",
+            "enableAutoUpgrade": false
```

Comment:
we can remove enableAutoUpgrade since its false by default?

Reply:
Remember seeing logs getting flooded with upgrade error message if this flag is not set using some version of dev build hence added this flag explicitly.